src/events/mod.rs

```diff
@@ -844,7 +844,7 @@ impl<'a> BytesCData<'a> {
     }
 
     /// Gets content of this text buffer in the specified encoding
-    pub(crate) fn decode(&self) -> Result<Cow<'a, str>, EncodingError> {
+    pub fn decode(&self) -> Result<Cow<'a, str>, EncodingError> {
```

Comment:
Also need to change doc string, because it is currently outdated. Will be used an encoding of a reader from which this event was produced.